host/pyproject.toml

```diff
@@ -37,7 +37,7 @@ dependencies = [
     "tabulate",
     "prompt_toolkit",
     "pygreat",
-    "ipython~=8.12",  # final version supported by Python 3.8
+    "ipython~=8.12",
```

Comment:
I think this should drop the version requirement too? (not just the comment about it)